docs/source/ProgrammingGuide/Compiling.md

```diff
@@ -129,17 +129,6 @@ There are numerous device backends, options, and architecture-specific optimizat
 ```
 which activates the OpenMP backend. All the options controlling device backends, options, architectures, and third-party libraries (TPLs) are given in [CMake Keywords](../keywords).
 
-### Using generate_makefile.bash
-As an alternative to calling the cmake command directly, the generate_makefile.bash command can be used to configure the CMake build environment.  The generate_makefile.bash equivalent to the above OpenMP example is as follows:
-
-```bash
-> ${srcdir}/generate_makefile.bash --compiler=g++ \
-  --with-openmp --prefix=${my_install_folder}
-```
-For a full list of generate_makefile.bash options use the command
-```bash
-> ${srcdir}/generate_makefile.bash --help
-```
 
 ### Spack
 An alternative to manually building with CMake is to use the Spack package manager.
```

Comment:
Maybe provide a link to Spack (git@github.com:spack/spack.git)?

Reply:
This is besides the scope of this PR

Reply:
Why?

Reply:
because this PR is about not advertising the `generate_makefile.bash` script and it has nothing to do with Spack

Reply:
The point of this PR is to move people away from Makefile use, and proposing alternatives (e.g., Spack) to CMake manual builds, n'est-ce pas?  How would those words sum to an advert for Makefile?